crates/forge/src/test_filter.rs

```diff
@@ -69,6 +78,13 @@ impl TestsFilter {
             NameFilter::ExactMatch(name) => {
                 cases.into_iter().filter(|tc| tc.name == *name).collect()
             }
+            NameFilter::OnlyFailed => match Self::read_test_fails_file() {
+                Ok(result) => cases
+                    .into_iter()
+                    .filter(|tc| result.iter().any(|name| name == &tc.name))
+                    .collect(),
+                Err(_) => cases,
```

Comment:
I wonder if we should output some information like: "Unable to find previously failed tests, re-running all.". Because in the way you wrote it it will always fail silently with no feedback for the user.

Reply:
In the way I wrote in the case when in some cases the filter doesn't work it will run all cases, and generate the new failed file.  I'm wondering what is the best solution in the case of error 🤔

Reply:
It shouldn't fail silently in any scenario, we should just bubble the error from `read_tests_failed_file`